core/test/audits/prioritize-lcp-image-test.js

```diff
@@ -8,13 +8,15 @@ import PrioritizeLcpImage from '../../audits/prioritize-lcp-image.js';
 import {networkRecordsToDevtoolsLog} from '../network-records-to-devtools-log.js';
 import {createTestTrace} from '../create-test-trace.js';
 
-const rootNodeUrl = 'http://example.com:3000';
-const mainDocumentNodeUrl = 'http://www.example.com:3000';
-const scriptNodeUrl = 'http://www.example.com/script.js';
+const requestedUrl = 'http://example.com:3000';
+const mainDocumentUrl = 'http://www.example.com:3000';
+const finalDisplayedUrl = 'http://www.example.com:3000/some-page.html';
```

Comment:
the test was using `mainDocumentUrl` for all the URLs, even though the networkRecords had a redirect from a `requestedUrl` to a different `mainDocumentUrl`/`finalDisplayedUrl`. Made it match the network records and be a bit more explicit, as well as adding a different `finalDisplayedUrl` to make it clear that it's not using that in any of the audit results.